tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -6696,6 +6696,14 @@ def _is_disagg_inflight_cancel_active(self) -> bool:
             self._disagg_inflight_cancel_unsupported_logged = True
         return False
 
+    def _request_remote_kv_transfer_abort(self, request: LlmRequest) -> None:
+        """Best-effort abort request to the peer; failures are retried later."""
+        try:
+            self.kv_cache_transceiver.request_remote_abort(request)
+        except Exception as error:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)